Additional allowlist rules for locked-down kubernetes environments (#27604)
DataDog/documentation: content/en/agent/configuration/network.md

To avoid running out of storage space, the Agent stores the metrics on disk only if the total storage space used is less than 80 percent. This limit is defined by `forwarder_storage_max_disk_ratio` setting.

> ## Installing the Operator

Reviewer: ```suggestion
## Installing the Datadog Operator
```

> If you are installing the Datadog Operator in Kubernetes environment with limited connectivity, you also want to allowlist the following endpoints for TCP port 443, based on your location:

Reviewer: ```suggestion
If you are installing the Datadog Operator in a Kubernetes environment with limited connectivity, you need to allowlist the following endpoints for TCP port 443, based on your location:
```
